### PR TITLE
Add team

### PR DESCRIPTION
**Pull request by miguelbrandao**

### CODE REVIEW

**Comment by alves-luis** [comment]
I think there's too much padding at the top and bottom of the "StyledBox".

Also, given that the TopSection component has changed, a lot of components who relied on it are not using the background image, as there isn't a default one (noticed the countdown, and the top of the pages)

File: data/team.json [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+        "facebook": "",
+        "twitter": "",
+        "github": "manilator",
+        "linkedin": "",
+        "img": "miguel_solino.jpg"
+      }
+    ]
+  },
+  {
+    "team": "Apoios e Patrocínios",
```
Comment (nelsonmestevao) [inline data/team.json]: It's missing Rafaela Pinho here!

Comment (miguelbrandao) [reply data/team.json]: Because she doesn't have a pic AFAIK

Comment (nelsonmestevao) [reply data/team.json]: Add a placeholder but it should be there

Comment (nelsonmestevao) [reply data/team.json]: Anyway, this page can be delayed until we have her picture

**Comment by miguelbrandao** [comment]
Closed by #96

**Pull request closed without merging**